cocos/ui/UIEditBox/UIEditBoxImpl-common.h

```diff
@@ -133,6 +136,14 @@ class EditBoxImplCommon : public EditBoxImpl
     
     int   _maxLength;
     Size _contentSize;
+    
```

Comment:
why duplicate these values both in `EditBox.h` and `EditBox-common.h`?
I think `EditBox-common.h` is the implementation details, it could use the data in `EditBox.h`, right?